docs/docsite/rst/become.rst

```diff
@@ -269,15 +272,29 @@ The following are some ways to use ``become`` with an admin token:
       win_reboot:
       when: uac_result|changed
 
+Local Service Accounts
+----------------------
+
+Prior to Ansible version 2.5, ``become`` only worked with a local or domain
+user account. Local service accounts like ``System`` or ``NetworkService``
+could not be used as ``become_user`` in these older versions. This restriction
+has been lifted since the 2.5 release of Ansible. The three service accounts
+that can be set under ``become_user`` are
+
+* System
+* NetworkService
+* LocalService
+
+Local service accounts do not have a password associated with its own account
```

Comment:
Please revise to: “Because local service accounts do not have passwords, the ``ansible_become_password`` parameter is not required and is ignored if specified.”